spec/controllers/ansible_credential_controller_spec.rb

```diff
@@ -0,0 +1,34 @@
+describe AnsibleCredentialController do
+  include CompressedIds
+
+  before do
+    EvmSpecHelper.create_guid_miq_server_zone
+    login_as FactoryGirl.create(:user_admin)
+  end
+
+  context "#show" do
+    let(:machine_credential) { FactoryGirl.create(:embedded_machine_credential, :options=>{}) }
```

Comment:
Found correct factory `:embedded_ansible_scm_credential` and it works for me. Please try it out :)